core/modules/system/css/system.theme.css

```diff
@@ -12,6 +12,13 @@ fieldset {
   margin-bottom: 1em;
   padding: 0.5em;
 }
+details summary {
```

Comment:
In Basis the fieldset summary is ALL CAPS. Also Basis overrides `system.theme.css` so this CSS doesnt run. The summary gets apointer in Basis though, because its a hyperlink. 

So short version: in Basis the CSS here in this PR doesnt work, and therefore looks different from fieldset summaries. We could simply add this CSS to Basis.

Reply:
I added these bits to Basis in `base.css`